perfkitbenchmarker/disk.py

```diff
@@ -53,6 +53,8 @@
 REGION = 'region'
 LEGACY_DISK_TYPE = 'legacy_disk_type'
 
+WINDOWS = 'windows'
```

Comment:
I actually tried referencing the one in windows_virtual_machine.WindowsMixin.OS_TYPE, but got a recursive import. I think benchmark_spec.py would have the same problem. I agree that we should move the os_type constants into a separate module.